src/Tomcat/TomcatController.ts

```diff
@@ -248,21 +258,21 @@ export class TomcatController {
                         needRestart = true;
                     } catch (err) {
                         console.error(err.toString());
-                        vscode.window.showErrorMessage(localize('tomcatExt.stopFailure', 'Failed to stop Tomcat Server {0}', serverInfo.getName()));
+                        vscode.window.showErrorMessage(localize('tomcatExt.stopFailure', 'Failed to stop Tomcat Server {0}', serverName));
                     }
                 }
             });
 
-            const javaProcss: Promise<void> = Utility.executeCMD(output, 'java', { shell: true }, ...args);
-            this.startDebugSession(debugPort, workspaceFolder);
-            await javaProcss;
+            const javaProcess: Promise<void> = Utility.executeCMD(output, 'java', { shell: true }, ...this.getJavaArgs(serverInfo, true));
```

Comment:
how about `await Utility.executeCMD(output, 'java', { shell: true }, ...this.getJavaArgs(serverInfo, true));` then start the debug session?